linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpLoggerConfig.scala

```diff
@@ -0,0 +1,29 @@
+package io.buoyant.linkerd.protocol
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.{Filter, Service, ServiceFactory, Stack, Stackable}
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.util.Future
+import io.buoyant.config.PolymorphicConfig
+
+abstract class HttpLoggerConfig extends PolymorphicConfig {
+  @JsonIgnore
+  def mk(): Filter[Request, Response, Request, Response]
```

Comment:
probably want to add `params: Stack.Params` as a parameter to `mk` so that the filter has access to the stack params (like dst, and endpointAddr, for example).